python/call_synthesize.py

```diff
@@ -31,16 +32,24 @@ def call_synthesize(args, text):
     if args.session_id:
         metadata = [('session_id', args.session_id)]
 
+    apl = None
+    if args.play:
+        apl = AudioPlayer(
+            sample_rate_hertz=int(args.sample_rate),
+            encoding=args.audio_encoding
+        )
     asv = AudioSaver()
     asv.setEncoding(audio_encoding)
 
     try:
         if args.response == "streaming":
-            internal_synthesize_streaming(stub, request, timeout, metadata, asv)
+            internal_synthesize_streaming(stub, request, timeout, metadata, asv, apl)
         elif args.response == "single":
-            internal_synthesize(stub, request, timeout, metadata, asv)
+            internal_synthesize(stub, request, timeout, metadata, asv, apl)
         else:
             raise RuntimeError("Unsupported response type: " + args.response)
+        if args.play:
+            apl.stop()
```

Comment:
shouldn't you change this code to wit apl:
and and exception handling there with apl.stop() ?

Reply:
I've decided it would be harder to read (`audioPlayer` is not initialized if it's not used) but I made some changes to enforce that `audioPlayer` will always be stopped.